robot/lib/Src/drivers/ICM20948.cpp

```diff
@@ -206,8 +202,9 @@ uint8_t ICM20948::read_register(uint8_t bank, uint8_t address) {
     return data;
 }
 
-<template T>
-std::vector<T> ICM20948::burst_read(uint8_t bank, uint8_t address, uint8_t length) {
+void ICM20948::burst_read(uint8_t bank, uint8_t address, uint8_t* buffer_out, size_t length) {
```

Comment:
Not sure if intentional, but burst reads aren't being used in the read* (eg accel_x) type calls you expect them to be used in